content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,221 +11,196 @@ weight: 10
 
-{{< note >}}
-You must have your own Ceph server running with the share exported before you can use it.
-{{< /note >}}
+The CSI was introduced to standardize connectivity to externally managed volumes, such as cloud platform storage. However, Kubernetes still includes many volume types for connecting to specific cloud storage providers. 
 
-See the [CephFS example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/volumes/cephfs/) for more details.
+External Volume Types:
+   * [awsElasticBlockStore](#awselasticblockstore)
+   * [azureDisk](#azuredisk)
+   * [azureFile](#azurefile)
+   * [cinder](#cinder)
+   * [flocker](#flocker)
+   * [gcePersistentDisk](#gcepersistentdisk)
+   * [portworxVolume](#portworxvolume)
+   * [quobyte](#quobyte)
+   * [storageos](#storageos)
+   * [scaleIO](#scaleio)
+   * [vsphereVolume](#vspherevolume)
 
-### cinder
+## Specify subPaths on volumes {#using-subpath}
 
-{{< note >}}
-Kubernetes must be configured with the OpenStack cloud provider.
-{{< /note >}}
+Sometimes, it is useful to share one volume for multiple uses in a single pod. The `volumeMounts.subPath` property specifies a sub-path inside the referenced volume instead of its root.
 
-The `cinder` volume type is used to mount the OpenStack Cinder volume into your pod.
+The following example shows how to configure a Pod with a LAMP stack (Linux Apache MySQL PHP)
+using a single, shared volume. This sample `subPath` configuration is not recommended
+for production use.
 
-#### Cinder volume configuration example
+The PHP application's code and assets map to the volume's `html` folder and
+the MySQL database is stored in the volume's `mysql` folder. For example:
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: test-cinder
+  name: my-lamp-site
 spec:
-  containers:
-  - image: k8s.gcr.io/test-webserver
-    name: test-cinder-container
-    volumeMounts:
-    - mountPath: /test-cinder
-      name: test-volume
-  volumes:
-  - name: test-volume
-    # This OpenStack volume must already exist.
-    cinder:
-      volumeID: "<volume id>"
-      fsType: ext4
+    containers:
+    - name: mysql
+      image: mysql
+      env:
+      # password in config yaml for testing only
+      # use a Kubernetes Secret in production
+      - name: MYSQL_ROOT_PASSWORD
+        value: "rootpasswd"
+      volumeMounts:
+      - mountPath: /var/lib/mysql
+        name: site-data
+        subPath: mysql
+    - name: php
+      image: php:7.0-apache
+      volumeMounts:
+      - mountPath: /var/www/html
+        name: site-data
+        subPath: html
+    volumes:
+    - name: site-data
+      persistentVolumeClaim:
+        claimName: my-lamp-site-data
 ```
 
-#### OpenStack CSI migration
+Note: The password is defined in the config yaml, which is insecure. See Kubernetes Secrets for a secure solution.
 
-{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+## Core Volume Types
```

Comment:
```suggestion
## Built-in volume types
```

The full list of built-in volume types could move to the reference section; for example, in a follow-up PR.